tests/_support/AcceptanceTester.php

```diff
@@ -12,15 +12,216 @@
  * @method void am($role)
  * @method void lookForwardTo($achieveValue)
  * @method void comment($description)
- * @method \Codeception\Lib\Friend haveFriend($name, $actorClass = NULL)
+ * @method \Codeception\Lib\Friend haveFriend($name, $actorClass = null)
  *
  * @SuppressWarnings(PHPMD)
-*/
+ */
 class AcceptanceTester extends \Codeception\Actor
```

Comment:
our AcceptanceTester is starting to grow. Instead I would suggest that you start moving the methods into Pages and Steps.

https://github.com/Codeception/Codeception/blob/master/docs/07-BDD.md#configuration

See example at https://github.com/joomla-projects/gsoc16_browser-automated-tests/pull/15/files#diff-0a4914515e2c67ed1abfc54b96e7df28R39